devfile.yaml

```diff
@@ -0,0 +1,25 @@
+schemaVersion: 2.1.0
+metadata:
+  name: catalog-spring-boot
+attributes:
+  che-theia.eclipse.org/sidecar-policy: mergeImage
+  metadata-name-field: generateName
+  metadata-name-original-value: catalog-spring-boot
+projects:
+  - git:
+      remotes:
+        origin: 'https://github.com/evanshortiss/catalog-spring-boot.git'
+    name: catalog-spring-boot
+components:
+  - container:
+      env:
+        - name: CHE_DASHBOARD_URL
+          value: 'https://devspaces.apps.sandbox-m2.ll9k.p1.openshiftapps.com'
+        - name: CHE_PLUGIN_REGISTRY_URL
+          value: 'https://devspaces.apps.sandbox-m2.ll9k.p1.openshiftapps.com/plugin-registry/v3'
+        - name: CHE_PLUGIN_REGISTRY_INTERNAL_URL
+          value: 'http://plugin-registry.crw.svc:8080/v3'
+      image: 'quay.io/devfile/universal-developer-image:ubi8-38da5c2'
+      memoryLimit: 512M
```

Comment:
This line is the only change necessary in the autogenerated devfile, so we could also have the attendees make this change themselves.

Reply:
My personal preference would be to change this as you suggest. Every "friction point" we can resolve makes sense I think and lab participants have hopefully an easier time to run through then :)

Reply:
Just verified this fix in my environment and the mvn build runs fine. Except it takes really long in my case. In particular dependency downloads are very slow it seems. Should we note in the lab content that this build might take several minutes?

Reply:
yes good idea. We don't have afaik any proxy to speed up the download

Reply:
I didn't find the build slow when I ran it, but maybe I was lucky?

@blues-man regarding the hardcoding. I assume that setting CHE_DASHBOARD_URL to "https://devspaces.apps.sandbox-m2.ll9k.p1.openshiftapps.com" does not make sense, since someone might run this devfile on a different cluster. I am unsure if it's safe to omit this value.

Reply:
See this one https://github.com/modernizing-java-applications-book/catalog-spring-boot/pull/2, closing with PR as it is not needed anymore. I've added a Spring Boot devfile and Mario added the new dependency structure, thanks!